ai-missions/high-risk-flu-shot-mission/skills/send_engage_sydneycare_chatbot_notification/Readme.md

```diff
@@ -0,0 +1,34 @@
+### Send Sydneycare Chatbot Notification
+
+#### Description:
+For the given profile IDs, this skill sends notifications to members for engaging with a chatbot in SydneyCare to discuss flu shot options.
```

Comment:
Change to: For the Profile IDs derived from the configured Cohort, notifications are sent to members instructing them to engage with a chatbot in the SydneyCare app to discuss flu shot options.